shared/tracker/index.js

```diff
@@ -131,6 +131,7 @@ export default connect(
   },
   (stateProps, dispatchProps, ownProps) => {
     return {
+      proofs: [],
```

Comment:
I was following and unfollowing jzila over and over

On Thursday, March 3, 2016, Marco Munizaga <notifications@github.com> wrote:

> In shared/tracker/index.js
> <https://github.com/keybase/client/pull/2204#discussion_r54973166>:
>
> > @@ -131,6 +131,7 @@ export default connect(
> >    },
> >    (stateProps, dispatchProps, ownProps) => {
> >      return {
> > +      proofs: [],
>
> How can I repro? I tried a couple trackers and they all seem to have this
> state.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/2204/files#r54973166>.
>